.openshift-ci/scripts/lib.sh

```diff
@@ -140,3 +140,91 @@ get_pr_details() {
     _PR_DETAILS="$pr_details"
     echo "$pr_details"
 }
+
+import_creds() {
+    shopt -s nullglob
+    for cred in /tmp/secret/**/[A-Z]*; do
+        export "$(basename "$cred")"="$(cat "$cred")"
+    done
+}
+
+ensure_secret_exists() {
+    local secret_name="$1"
+
+    if [[ -z "${secret_name}" ]]; then
+        import_creds
```

Comment:
Does it mean as a side effect everything will be already exported? If so, probably makes sense to mention in the commentary, in case if in the future this function will be used in some other contexts, what do you think?

Reply:
This is fine as is right now, but maybe we want an alternative to `import_creds` that imports just the secret that we are asking for? I think it could make this a function a little easier to follow, at least it took me a second to realize that `-z "${secret_name}"` is a different check in 157 and 159, (also, lazzy initialization and all that jazz).